website/docs/r/monitor_metric_alert.html.markdown

```diff
@@ -106,6 +106,8 @@ An `action` block supports the following:
 * `action_group_id` - (Required) The ID of the Action Group can be sourced from [the `azurerm_monitor_action_group` resource](./monitor_action_group.html)
 * `webhook_properties` - (Optional) The map of custom string properties to include with the post operation. These data are appended to the webhook payload.
 
+-> **NOTE** In order to add additional action groups. you must use multiple blocks.
```

Comment:
This appears a bit redundant since this is specified in a different manner on this line here:
https://github.com/hashicorp/terraform-provider-azurerm/blob/8aec850b74aab13939ae87f9085f39f90c524d19/website/docs/r/monitor_metric_alert.html.markdown?plain=1#L85

Could you provide some more background on what this is supposed to clarify?

Reply:
At first glance, it may not be clear, having the note serves as clarification.